source/api_cc/include/common.h

```diff
@@ -19,6 +19,8 @@
 namespace deepmd {
 
 typedef double ENERGYTYPE;
+// TODO: currently we only implement TF; reserve for future use
+enum Backend { TENSORFLOW, PYTORCH, PADDLE, UNKNOWN };
```

Comment:
Consider using a more descriptive name for the enum `Backend`. Something like `DeepPotBackend` would be more clear and less likely to conflict with other names in the codebase. [medium]

Reply:
Rename to `DPBackend`